src/components/NcRichText/NcReferenceList.vue

```diff
@@ -146,11 +148,8 @@ export default {
 }
 </script>
 <style lang="scss" scoped>
-.widget--list {
+.widgets--list {
 	width: var(--widget-full-width, 100%);
```

Comment:
This breaks text styling as we set a custom full width to span the whole page, however the actual styling is applied in the NcReferenceWidget component.

This should fix it:

```suggestion
.widgets--list {
	width: 100%;
```

Reply:
yup, wasn't noticed before of wrong classname, probably?

Reply:
Yes, I guess so